misc/greeter/failing_client.py

```diff
@@ -45,17 +45,17 @@ async def worker(port, num_concurrent_streams, num_requests_per_stream,
             raise ValueError(f"Unknown load type: {load_type}")
         for idx in range(num_rounds):
             start = time.time()
-            task_results = anyio.create_queue(sys.maxsize)
+            send_queue, receive_queue = anyio.create_memory_object_stream(max_buffer_size=sys.maxsize)
```

Comment:
Is it wise to have such a huge buffer?

Reply:
A buffer size of 0 or 1 should be okay. This only needs to be revisited if it doesn't work with those values.

Reply:
I'd have to confirm performance just in case.

Maybe another PR-- I left a TODO for now.